cmd/kogito/command/context/logger.go

```diff
@@ -22,15 +22,30 @@ import (
 
 var (
 	commandOutput io.Writer
+	outputFormat  string
 	logVerbose    bool
 )
 
 // GetDefaultLogger retrieves the default logger
 func GetDefaultLogger() *zap.SugaredLogger {
-	return getDefaultLoggerWithOut(logVerbose, commandOutput)
+	return getDefaultLoggerWithOut(logVerbose, outputFormat, commandOutput)
 }
 
 // GetDefaultLoggerWithOut returns a logger set to a given output
-func getDefaultLoggerWithOut(verbose bool, commandOutput io.Writer) *zap.SugaredLogger {
-	return logger.GetLoggerWithOptions("kogito-cli", &logger.Opts{Output: commandOutput, Verbose: verbose, Console: true})
+func getDefaultLoggerWithOut(verbose bool, outputFormat string, commandOutput io.Writer) *zap.SugaredLogger {
+	var badOutputFormatMsg string
+	if len(outputFormat) > 0 && outputFormat != "json" {
+		badOutputFormatMsg = "'" + outputFormat + "' is not a supported output format"
+		outputFormat = ""
+	}
+	logger := logger.GetLoggerWithOptions("kogito-cli", &logger.Opts{
+		Output:       commandOutput,
+		OutputFormat: outputFormat,
+		Verbose:      verbose,
+		Console:      true,
+	})
+	if len(badOutputFormatMsg) > 0 {
+		logger.Warn(badOutputFormatMsg)
```

Comment:
Why aren't you logging the warning directly into the first `if` ?

Reply:
Because I need to wait until I have created a logger with a corrected (cleared) output format first. Then I'll have a good logger to log the warning with (and with the bad output format name).

Reply:
yeah ... right ... sry ...

Reply:
No worries. :)